translations/Indonesian/general-rules.md

```diff
@@ -0,0 +1,26 @@
+# General Rules
+
+
+**Should translations be formal or informal?**
+Translations should be formal.

```

Comment:
Can you give a reason why (e.g. "other social networks use formal style), and an example?